source/ProtonPack/ProtonPack.ino

```diff
@@ -283,8 +296,76 @@ void loop() {
         }
       }
 
-      // Play a little bit of smoke and n-filter vent lights while firing. Just a tiny bit....
+      // Play a little bit of smoke and n-filter vent lights while firing and other misc sound effects.
       if(b_wand_firing == true) {
+        // Mix some impact sound effects.
+        if(ms_firing_sound_mix.justFinished() && FIRING_MODE == PROTON && b_stream_effects == true) {
+          uint8_t i_random = 0;
+
+          switch(i_last_firing_effect_mix) {
+            case S_FIRE_SPARKS:
+              i_random = random(0,2);
+            break;
+
+            case S_FIRE_SPARKS_3:
+            case S_FIRE_SPARKS_4:
+              i_random = 3;
+            break;
+
+            case S_FIRE_SPARKS_5:
+              i_random = 2;
+            break;
+
+            case S_FIRE_SPARKS_2:
+              i_random = 1;
+            break;
+
+            default:
+              i_random = 3;
+            break;
+          }
+
+          unsigned int i_s_random = random(2,4) * 1000;
```

Comment:
Is there benefit of making this range a Configuration.h option to affect how wide the range of randomness might be? Also, is this essentially setting a time between 2-4 seconds?

Reply:
Yes between 2-4 seconds, but sometimes 12-14 at times. If set in the configuration then it wont be random on each cycle.